docs/ehrql-new-tutorial-1a.md

```diff
@@ -59,110 +39,35 @@ Here, we import two components of Data Builder:
 * `Dataset` as provided by the query language, to create a dataset
 * the `patients` table, which is one of several data tables that ehrQL gives access to
 
-!!! todo
-
-    Consider using snippets or code annotations instead of copy-pasting lines.
-
-!!! warning
-    For the purposes of the tutorial,
-    we are accessing tables from `databuilder.tables.examples.tutorial`.
-
-    Different backends support access to different tables.
-    To access tables on real backends,
-    refer to the [Contracts reference](contracts-reference.md)
-    (to be renamed)
-    to look up the correct import statement
-    that specifies the backend and table name.
-
-#### Give names to values using Python's assignment operator, `=`
-
-Like in many other languages,
-the `=` operator is used to assign a *value*, on the right-hand side,
-to a *name*, on the left-hand side.
-
-#### Create a `Dataset`
-
-A valid dataset definition must contain a dataset assigned to the name `dataset`.
+### Create a `Dataset`
 
-A usual first step in writing a dataset definition is to create this empty dataset.
+A valid dataset definition must contain a dataset assigned to the name `dataset`. Like many other programming languages, we use `=` to assign a value to a variable name. In this case, we have assigned `Dataset()` to the variable `dataset`. This creates an empty dataset. 
 In subsequent steps,
 we specify the data from the available data tables
 that we wish to add to the dataset.
 
-!!! warning
+### Find year of birth
+Next we define a year of birth. `date_of_birth` is in the patient table and therefore we can assign it to this new variable. We want to only capture the `year` of birth so we add the `.year` to the end of this variable assignment. 
 
-    The dataset used as the basis of an ehrQL query
-    is the dataset that is last to be given the name `dataset`.
+### Set population
+Finally we set the population of the dataset. We use the special method called `set_population()` and pass in the definition of the population. In this case, we want to use our previously created `year_of_birth` and say, if year of birth is equal to or greater than 2000, include in this dataset. 
 
-    A simple way to make your dataset definition clear is to
-    only use the name `dataset` for the query's dataset.
-
-!!! warning
-
-    Likewise, if you accidentally include multiple calls to `set_population()`,
-    it is the call that appears closest to the end of the dataset definition
-    that takes effect.
-
-    In future, calling `set_population()` more than once will cause a dataset definition to fail;
-    see the associated [Data Builder](https://github.com/opensafely-core/databuilder/issues/775) issue.
```

Comment:
thought: I feel like this issue does warrant a mention somewhere, even if not in the tutorial. It should get fixed eventually, but for the time being, it's useful to know.

Reply:
Agreed but I think we should put it in explainer-document